Cargo.toml

```diff
@@ -153,11 +152,10 @@ serde_json = { version = '1.0.132', default-features = false, features = [
 ] }
 syn = "2.0"
 tempfile = "3.10.1"
-thiserror = { version = "1.0.48" }
+thiserror = { version = "2.0", default-features = false }
 time = { version = "0.3.36", default-features = false }
 tokio = { version = "1.0", features = ["rt-multi-thread", "macros"] }
 uplc = { version = "1.1.6" }
-lazy_static = "1.4.0"
```

Comment:
not needed anymore, also can be replaced with [LazyLock](https://doc.rust-lang.org/std/sync/struct.LazyLock.html)

Reply:
Merci!